rhc-ose-ansible/roles/openshift-management/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+openshift_token:
+openshift_master_url: localhost
+openshift_login_insecure_flag: --insecure-skip-tls-verify=true
+openshift_login_insecure: False
```

Comment:
I was more talking about `openshift_login_insecure`  seems to be a variable set without any references to it.